west.yml

```diff
@@ -102,7 +102,7 @@ manifest:
       revision: dcfa04f4aaf30ae7c2dbf34a05a569ca1c04b2ce
       path: tools/net-tools
     - name: hal_nxp
-      revision: 56915d2942a3af7205b3c4991a8b87e915de7f59
+      revision: pull/68/head
```

Comment:
The hal PR is now merged, so you can update this to the new SHA

Reply:
thx, done